spec/flows/early_result_spec.rb

```diff
@@ -32,8 +32,47 @@
       fill_in_other_income_screen_with_friends_and_family
       fill_in_ineligible_gross_income_screen(choice: "Skip remaining questions")
       confirm_screen("check_answers")
+      expect(all(".govuk-summary-card__title").map(&:text))
+        .to eq(
+          ["Client age",
+           "Partner and passporting",
+           "Level of help",
+           "Type of matter",
+           "Type of immigration or asylum matter",
+           "Number of dependants",
+           "Employment status",
+           "Client employment income 1",
+           "Client benefits",
+           "Client other income"],
+        )
     end
 
+    # it "when I am u18 and change answers on aggregation on an EE check" do
+    #   start_assessment
+    #   fill_in_forms_until(:applicant)
+    #   fill_in_applicant_screen(partner: "No", passporting: "No")
+    #   fill_in_dependant_details_screen
+    #   fill_in_employment_status_screen(choice: "Employed or self-employed")
+    #   fill_in_income_screen(gross: "3000", frequency: "Every month")
+    #   fill_in_forms_until(:other_income)
+    #   fill_in_other_income_screen
+    #   fill_in_ineligible_gross_income_screen(choice: "Skip remaining questions")
+    #   confirm_screen("check_answers")
+    #   expect(all(".govuk-summary-card__title").map(&:text))
+    #     .to eq(
+    #           ["Client age",
+    #            "Partner and passporting",
+    #            "Level of help",
+    #            "Type of matter",
+    #            "Type of immigration or asylum matter",
+    #            "Number of dependants",
+    #            "Employment status",
+    #            "Client employment income 1",
+    #            "Client benefits",
+    #            "Client other income"],
+    #           )
+    # end
+
```

Comment:
@willc-work did we mean to leave this commented out?

Reply:
removed now., this was to test if adding income values triggered the coverage, it doesnt